packages/l10n/package.json

```diff
@@ -29,18 +29,18 @@
     "@types/react": "^17.0.44",
     "lodash": "4.17.21",
     "moment": "^2.29.2",
+    "moment-timezone": "^0.5.34",
+    "prompts": "^2.4.2",
```

Comment:
Nit: this should be a dev dependency

Reply:
When I move `moment-timezone` to `devDependencies`, I get this error from `yarn build`:
```
🎁 error @commercetools-frontend/l10n "moment-timezone" is imported by "src/time-zone-information.ts" but the package is not specified in dependencies or peerDependencies
```

But I've added `prompts` to `devDependencies`